docs/content/preview/migrate/airgapped.md

```diff
@@ -0,0 +1,46 @@
+<!--
```

Comment:
Also include the steps to install docker image in an airgapped environment (as shared in the dms-docs channel).